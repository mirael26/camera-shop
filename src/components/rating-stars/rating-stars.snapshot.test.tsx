import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import RatingStars from './rating-stars';

test('RatingStars renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<RatingStars rating={4}/>);
  expect(asFragment()).toMatchSnapshot();
});
