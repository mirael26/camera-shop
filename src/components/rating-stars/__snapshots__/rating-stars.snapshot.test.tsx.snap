// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingStars renders correctly 1`] = `
<DocumentFragment>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      data-testid="full-star-icon"
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      data-testid="full-star-icon"
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      data-testid="full-star-icon"
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      data-testid="full-star-icon"
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
</DocumentFragment>
`;
